package.json

```diff
@@ -44,6 +44,7 @@
     "moment": "^2.14.1",
     "morgan": "~1.7.0",
     "node-simple-logger": "0.0.1",
+    "node-uuid": "^1.4.7",
```

Comment:
You probably only need UUID4. I recommend to use uuid4: https://www.npmjs.com/package/uuid4